### ISSUE TITLE
Small UI bug when no results found online

### ISSUE THREAD

**Issue by anleac** [benkaiser/stretto#163]
![image](https://user-images.githubusercontent.com/13091188/102522912-2af6f880-4097-11eb-9253-9634d3bdbbc9.png)

I think that the string "more found online" shows regardless of if there are more results found.

**Comment by benkaiser** [comment]
Yeah it updates as soon as we receive the results from iTunes (even if there are none). Fix here is to use a different string when there are 0 results
